app/assets/javascripts/max-attendances.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+    var fieldsCount,
+        maxFieldsCount = 2,
+        $addLink = $('a.add_nested_fields');
+
+    function toggleAddLink() {
```

Comment:
This defines a named function in the global namespace, right? I don't know if that is cool. But then again, I am not a JS person :8ball: :sweat_smile:

Come to think of it: are the vars `fieldsCount`, `maxFieldsCount`  and `$addlink` global as well?